### ISSUE TITLE
[Question]: UIE多块cpu训练问题

### ISSUE THREAD

**Issue by fuqiang-benz** [PaddlePaddle/PaddleNLP#4612]
### 请提出你的问题

您好请教一下 我在使用UIE自定义训练的时候 想用多块cpu训练 命令是：python -m paddle.distributed.launch --nproc_per_node=3 --backend=gloo  ./finetune.py --device cpu --logging_steps 10 --save_steps 100 --eval_steps 100 --seed 1000 --output_dir ./checkpoint/model_best --per_device_train_batch_size 16 --per_device_eval_batch_size 16 --num_train_epochs 20 --learning_rate 1e-05 --do_train --do_eval --do_export --overwrite_output_dir  --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end True --save_total_limit 1 报错：命令是写着起3个进程一起训练，但是问题是 一直再启进程，启了好几十个进程，直到内存跑满然后服务器死机了（年前还可以跑，paddlenlp更新之后就用不了了） 我用的代码就是develp分支下的 信息抽取功能，麻烦帮忙看一下辛苦
![微信截图_20230201143239](https://user-images.githubusercontent.com/64388539/215968805-07b5f579-b0f5-43e6-a0ff-d412eae18b44.png)

**Comment by linjieccc** [comment]
@fuqiang-benz 你好，

有具体的报错信息么，从微调的命令来看没有指定训练集、验证集的路径

**Comment by fuqiang-benz** [comment]
> @fuqiang-benz 你好，
> 
> 有具体的报错信息么，从微调的命令来看没有指定训练集、验证集的路径

您好 没有具体的报错信息，就是会一直在启动进程 会启动很多进程直到内存跑满 我记得paddlepaddle2.3的时候还可以用，更新到2.4就不可以用了  训练代码finetune.py 代码里面已经指定训练集和验证集的路径了

**Comment by linjieccc** [comment]
我这边用paddle2.4.1似乎没有复现出你的问题，请问文档里面默认的例子可以跑通吗？
